SimplenoteUITests/AssertGeneric.swift

```diff
@@ -5,6 +5,7 @@ let notExpectedEnding = " is NOT as expected",
     notAbsentEnding = " NOT absent"
 
 let inAllNotesEnding = " in \"All Notes\"",
+    inNoteListEnding = " in \"Note List\"",
```

Comment:
Seeing this in an isolated diff made me notice the escaped `"`. 

There's a relatively recent Swift feature called [_Extended Strings Delimiters_](https://docs.swift.org/swift-book/LanguageGuide/StringsAndCharacters.html#ID606) which simplifies this syntax:

```suggestion
    inNoteListEnding = #" in "Note List""#,
```

Reply:
Thank you! 🦉